agent/ecs_client/model/api/api-2.json

```diff
@@ -961,7 +961,8 @@
         "journald",
         "gelf",
         "fluentd",
-        "awslogs"
+        "awslogs",
+        "logentries"
```

Comment:
Can you undo the change here?  This file is generated from a different one we have internally.  Once we get this merged, I can make the change internally.